src/box/alter.cc

```diff
@@ -244,6 +244,12 @@ index_opts_decode(struct index_opts *opts, const char *map,
 			 "less than or equal to 1");
 		return -1;
 	}
+	if (opts->lookup_cost_coeff <= 0 || opts->lookup_cost_coeff > 1) {
```

Comment:
Why is it [0; 1]? AFAIC, it can be any positive integer.

Reply:
Discussed it.
The motivation of using [0; 1] instead of [0; MAX_R_VALUE] is that:
- MAX_R_VALUE is calculated dynamically from the number of runs in the range
- It will be more comfortable for user to choose value in [0;1] range